Native compilation fails with unresolved type during parsing

### Describe the bug

I have a CLI project that is using the following extensions:

- quarkus-picocli
- quarkus-qute
- quarkus-rest-client-reactive-jackson

It also depends on:
- quarkus-arc
- quarkus-devtools-registry-client

When I'm compiling the project to native using `mvn package -Dnative` it fails.



### Expected behavior

Native compilation should work.

### Actual behavior

Build fails with the following exception:
```
Error: Unsupported features in 4 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.aayushatharva.brotli4j.encoder.Encoder$Parameters. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.BrotliEncoder.<init>(BrotliEncoder.java:53)
Call path from entry point to io.netty.handler.codec.compression.BrotliEncoder.<init>(Object):
	at io.netty.handler.codec.compression.BrotliEncoder.<init>(BrotliEncoder.java:52)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.newContentCompressor(CompressorHttp2ConnectionEncoder.java:263)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.newCompressor(CompressorHttp2ConnectionEncoder.java:333)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.writeHeaders(CompressorHttp2ConnectionEncoder.java:206)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.handleServerHeaderDecodeSizeError(Http2ConnectionHandler.java:741)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.onStreamError(Http2ConnectionHandler.java:717)
	at io.vertx.core.http.impl.VertxHttp2ConnectionHandler.onStreamError(VertxHttp2ConnectionHandler.java:145)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.onError(Http2ConnectionHandler.java:634)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.flush(Http2ConnectionHandler.java:193)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.aayushatharva.brotli4j.encoder.Encoder. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:79)
Call path from entry point to io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(ChannelHandlerContext, ByteBuf, boolean):
	at io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:73)
	at io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:33)
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:105)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.github.luben.zstd.Zstd. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ZstdEncoder.java:107)
Call path from entry point to io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ChannelHandlerContext, ByteBuf, boolean):
	at io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ZstdEncoder.java:91)
	at io.netty.handler.codec.compression.ZstdEncoder.flush(ZstdEncoder.java:164)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.github.luben.zstd.Zstd. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ZstdEncoder.java:143)
Call path from entry point to io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ByteBuf):
	at io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ZstdEncoder.java:138)
	at io.netty.handler.codec.compression.ZstdEncoder.flush(ZstdEncoder.java:165)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)

com.oracle.svm.core.util.UserError$UserException: Unsupported features in 4 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.aayushatharva.brotli4j.encoder.Encoder$Parameters. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.BrotliEncoder.<init>(BrotliEncoder.java:53)
Call path from entry point to io.netty.handler.codec.compression.BrotliEncoder.<init>(Object):
	at io.netty.handler.codec.compression.BrotliEncoder.<init>(BrotliEncoder.java:52)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.newContentCompressor(CompressorHttp2ConnectionEncoder.java:263)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.newCompressor(CompressorHttp2ConnectionEncoder.java:333)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.writeHeaders(CompressorHttp2ConnectionEncoder.java:206)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.handleServerHeaderDecodeSizeError(Http2ConnectionHandler.java:741)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.onStreamError(Http2ConnectionHandler.java:717)
	at io.vertx.core.http.impl.VertxHttp2ConnectionHandler.onStreamError(VertxHttp2ConnectionHandler.java:145)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.onError(Http2ConnectionHandler.java:634)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.flush(Http2ConnectionHandler.java:193)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.aayushatharva.brotli4j.encoder.Encoder. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:79)
Call path from entry point to io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(ChannelHandlerContext, ByteBuf, boolean):
	at io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:73)
	at io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:33)
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:105)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.github.luben.zstd.Zstd. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ZstdEncoder.java:107)
Call path from entry point to io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ChannelHandlerContext, ByteBuf, boolean):
	at io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ZstdEncoder.java:91)
	at io.netty.handler.codec.compression.ZstdEncoder.flush(ZstdEncoder.java:164)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.github.luben.zstd.Zstd. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ZstdEncoder.java:143)
Call path from entry point to io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ByteBuf):
	at io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ZstdEncoder.java:138)
	at io.netty.handler.codec.compression.ZstdEncoder.flush(ZstdEncoder.java:165)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)

	at com.oracle.svm.core.util.UserError.abort(UserError.java:87)
	at com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:233)
	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:759)
	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:529)
	at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:488)
	at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
	at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:569)
	at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:122)
	at com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus.main(NativeImageGeneratorRunner.java:599)
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Unsupported features in 4 methods
Detailed message:
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.aayushatharva.brotli4j.encoder.Encoder$Parameters. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.BrotliEncoder.<init>(BrotliEncoder.java:53)
Call path from entry point to io.netty.handler.codec.compression.BrotliEncoder.<init>(Object):
	at io.netty.handler.codec.compression.BrotliEncoder.<init>(BrotliEncoder.java:52)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.newContentCompressor(CompressorHttp2ConnectionEncoder.java:263)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.newCompressor(CompressorHttp2ConnectionEncoder.java:333)
	at io.netty.handler.codec.http2.CompressorHttp2ConnectionEncoder.writeHeaders(CompressorHttp2ConnectionEncoder.java:206)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.handleServerHeaderDecodeSizeError(Http2ConnectionHandler.java:741)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.onStreamError(Http2ConnectionHandler.java:717)
	at io.vertx.core.http.impl.VertxHttp2ConnectionHandler.onStreamError(VertxHttp2ConnectionHandler.java:145)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.onError(Http2ConnectionHandler.java:634)
	at io.netty.handler.codec.http2.Http2ConnectionHandler.flush(Http2ConnectionHandler.java:193)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.aayushatharva.brotli4j.encoder.Encoder. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:79)
Call path from entry point to io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(ChannelHandlerContext, ByteBuf, boolean):
	at io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:73)
	at io.netty.handler.codec.compression.BrotliEncoder.allocateBuffer(BrotliEncoder.java:33)
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:105)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.github.luben.zstd.Zstd. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ZstdEncoder.java:107)
Call path from entry point to io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ChannelHandlerContext, ByteBuf, boolean):
	at io.netty.handler.codec.compression.ZstdEncoder.allocateBuffer(ZstdEncoder.java:91)
	at io.netty.handler.codec.compression.ZstdEncoder.flush(ZstdEncoder.java:164)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)
Error: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: com.github.luben.zstd.Zstd. To diagnose the issue you can use the --allow-incomplete-classpath option. The missing type is then reported at run time when it is accessed the first time.
Trace:
	at parsing io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ZstdEncoder.java:143)
Call path from entry point to io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ByteBuf):
	at io.netty.handler.codec.compression.ZstdEncoder.flushBufferedData(ZstdEncoder.java:138)
	at io.netty.handler.codec.compression.ZstdEncoder.flush(ZstdEncoder.java:165)
	at io.netty.channel.AbstractChannelHandlerContext.invokeFlush0(AbstractChannelHandlerContext.java:750)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:765)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071)
	at java.lang.Thread.run(Thread.java:829)
	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:596)
	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
	at com.oracle.svm.core.code.IsolateEnterStub.PosixJavaThreads_pthreadStartRoutine_e1f4a8c0039f8337338252cd8734f63a79b5e3df(generated:0)

	at com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:129)
	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:756)
	... 6 more
[rhbq-supported-extension-generator-1.0.0-SNAPSHOT-runner:16029]      [total]:  31 709,83 ms,  3,40 GB 
```

### How to Reproduce?

I can provide access to a private github repo on request, @gastaldi and @cescoffier are already invited

### Output of `uname -a` or `ver`

Darwin tqvarnst-mac 20.6.0 Darwin Kernel Version 20.6.0: Tue Oct 12 18:33:42 PDT 2021; root:xnu-7195.141.8~1/RELEASE_X86_64 x86_64

### Output of `java -version`

openjdk version "11.0.11" 2021-04-20 OpenJDK Runtime Environment AdoptOpenJDK-11.0.11+9 (build 11.0.11+9) OpenJDK 64-Bit Server VM AdoptOpenJDK-11.0.11+9 (build 11.0.11+9, mixed mode)

### GraalVM version (if different from Java)

openjdk version "11.0.13" 2021-10-19 OpenJDK Runtime Environment GraalVM CE 21.3.0 (build 11.0.13+7-jvmci-21.3-b05) OpenJDK 64-Bit Server VM GraalVM CE 21.3.0 (build 11.0.13+7-jvmci-21.3-b05, mixed mode, sharing)

### Quarkus version or git rev

2.6.3.Final<

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

This could be related to this commit: https://github.com/quarkusio/quarkus/commit/919b71c257e5ff853f158da1b033bbc701b7965c

--- a/extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/HttpContentCompressorSubstitutions.java
+++ b/extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/HttpContentCompressorSubstitutions.java
@@ -1,4 +1,4 @@
-package io.quarkus.vertx.http.runtime.graal;
+package io.quarkus.netty.runtime.graal;
 
 import static io.netty.handler.codec.http.HttpHeaderValues.DEFLATE;
 import static io.netty.handler.codec.http.HttpHeaderValues.GZIP;

